pm_asephonons/test-data/test_out.txt

```diff
@@ -7,284 +7,22 @@
-		5	-0.303347	 0.125695	 0.055356
-		6	-0.258705	-0.227629	-0.142736
-		7	-0.258705	 0.227629	 0.142736
-		8	 0.303347	 0.125695	-0.055356
-		Mode 16:
-		Atom x			 y			 z
-		1	 0.051837	-0.030659	-0.013696
-		2	-0.274591	 0.255684	 0.015966
-		3	 0.293128	 0.030659	-0.283773
-		4	 0.274591	 0.014393	-0.328998
-		5	-0.051837	-0.282373	 0.283773
-		6	 0.022877	-0.014393	-0.015966
-		7	-0.022877	-0.255684	 0.328998
-		8	-0.293128	 0.282373	 0.013696
-		Mode 17:
-		Atom x			 y			 z
-		1	-0.245922	 0.206385	-0.030241
-		2	-0.056507	 0.054725	-0.011078
-		3	 0.184125	-0.206385	 0.290356
-		4	 0.056507	-0.375323	 0.050719
-		5	 0.245922	-0.166744	-0.290356
-		6	-0.316622	 0.375323	 0.011078
-		7	 0.316622	-0.054725	-0.050719
-		8	-0.184125	 0.166744	 0.030241
-		Mode 18:
-		Atom x			 y			 z
-		1	 0.112310	 0.115555	-0.304919
-		2	-0.077046	-0.086728	 0.369011
-		3	 0.234545	-0.115555	-0.009228
-		4	 0.077046	-0.208962	 0.022156
-		5	-0.112310	 0.275611	 0.009228
-		6	 0.237101	 0.208962	-0.369011
-		7	-0.237101	 0.086728	-0.022156
-		8	-0.234545	-0.275611	 0.304919
-		Mode 19:
-		Atom x			 y			 z
-		1	 0.381481	 0.260338	 0.096655
-		2	-0.086133	-0.218379	 0.240651
-		3	-0.043466	-0.260338	 0.108466
-		4	 0.086133	 0.206568	-0.097363
-		5	-0.381481	-0.117049	-0.108466
-		6	-0.291254	-0.206568	-0.240651
-		7	 0.291254	 0.218379	 0.097363
-		8	 0.043466	 0.117049	-0.096655
-		Mode 20:
-		Atom x			 y			 z
-		1	-0.169255	 0.353999	 0.039909
-		2	-0.259399	 0.175095	 0.060531
-		3	-0.129669	-0.353999	-0.270696
-		4	 0.259399	 0.135509	 0.359455
-		5	 0.169255	 0.065987	 0.270696
-		6	-0.028612	-0.135509	-0.060531
-		7	 0.028612	-0.175095	-0.359455
-		8	 0.129669	-0.065987	-0.039909
-		Mode 21:
-		Atom x			 y			 z
-		1	-0.006825	 0.004638	 0.380772
-		2	 0.301270	 0.309453	 0.227764
-		3	 0.237612	-0.004638	 0.006303
-		4	-0.301270	 0.065016	-0.003023
-		5	 0.006825	 0.220103	-0.006303
-		6	-0.085805	-0.065016	-0.227764
-		7	 0.085805	-0.309453	 0.003023
-		8	-0.237612	-0.220103	-0.380772
-		Mode 22:
-		Atom x			 y			 z
-		1	-0.269887	 0.064854	 0.218986
-		2	 0.269887	-0.064854	-0.218986
-		3	-0.269887	 0.064854	 0.218986
-		4	 0.269887	-0.064854	-0.218986
-		5	-0.269887	 0.064854	 0.218986
-		6	 0.269887	-0.064854	-0.218986
-		7	 0.269887	-0.064854	-0.218986
-		8	-0.269887	 0.064854	 0.218986
-		Mode 23:
-		Atom x			 y			 z
-		1	-0.084811	 0.286300	-0.189314
-		2	 0.084811	-0.286300	 0.189314
-		3	-0.084811	 0.286300	-0.189314
-		4	 0.084811	-0.286300	 0.189314
-		5	-0.084811	 0.286300	-0.189314
-		6	 0.084811	-0.286300	 0.189314
-		7	 0.084811	-0.286300	 0.189314
-		8	-0.084811	 0.286300	-0.189314
-		Mode 24:
-		Atom x			 y			 z
-		1	-0.212056	-0.197045	-0.202991
-		2	 0.212056	 0.197045	 0.202991
-		3	-0.212056	-0.197045	-0.202991
-		4	 0.212056	 0.197045	 0.202991
-		5	-0.212056	-0.197045	-0.202991
-		6	 0.212056	 0.197045	 0.202991
-		7	 0.212056	 0.197045	 0.202991
-		8	-0.212056	-0.197045	-0.202991
\ No newline at end of file
+		Atom x			 y			 z
```

Comment:
Same thing here? It might make the `test_out.txt` to long though
```
Mode 1
Atom x			 y			 z
1
2
3
4
5
6
7
8
Mode 2
...
Mode 24
```

Reply:
Yep. Duplicate lines always match the first occurrence (i.e. no point including all the 12345678 for each mode), but I'll add all the non-duplicates